### DISCUSSION TITLE
Classic Explorer - how do I use / start it?

### DISCUSSION THREAD

**Discussion by TLorenzPH** [Open-Shell/Open-Shell-Menu#1974] [Q&A]
Hi.
I wonder how to use/start the classic explorer.
Open Shell menu is working on win11 but I'm not sure if I have_/can run the classic explorer.
I also have installed explorer patcher - it that tool maybe disabling/hiding the classic explorer?

Thomas

**Comment by MaximilianKohler** [comment]
View -> Options -> Classic Explorer bar.

**Comment by TLorenzPH** [comment]
Hi

In Win 11 there is not views.
And I do not see classic shell in the explorer….

***@***.***

Thomas Lorenz

Von: MaximilianKohler ***@***.***>
Gesendet: Donnerstag, 22. August 2024 11:45
An: Open-Shell/Open-Shell-Menu ***@***.***>
Cc: Thomas Lorenz ***@***.***>; Author ***@***.***>
Betreff: [EXT] Re: [Open-Shell/Open-Shell-Menu] Classic Explorer - how do I use / start it? (Discussion #1974)


View -> Options -> Classic Explorer bar.
